puzzlegenerator/puzzlegenerator_ui/src/main/java/com/flurnamenpuzzle/generator/ui/view/SaveTargetCard.java

```diff
@@ -0,0 +1,96 @@
+package com.flurnamenpuzzle.generator.ui.view;
+
+import java.awt.Dimension;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.io.File;
+
+import javax.swing.JButton;
+import javax.swing.JFileChooser;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JTextField;
+import javax.swing.border.EmptyBorder;
+import javax.swing.plaf.metal.MetalIconFactory.FolderIcon16;
+
+import net.miginfocom.swing.MigLayout;
+
+import com.flurnamenpuzzle.generator.ui.Observable;
+import com.flurnamenpuzzle.generator.ui.Observer;
+import com.flurnamenpuzzle.generator.ui.PuzzleGeneratorConfig;
+import com.flurnamenpuzzle.generator.ui.PuzzleGeneratorController;
+
+public class SaveTargetCard extends JPanel implements Observer{
```

Comment:
Das Naming gefällt mir nicht ganz.. Die Klasse steht ja eigentlich für einen Screen, auf dem man etwas auswählt (also auch eher eine ...Selection...Card) oder nicht? Ich würde an etwas wie TargetPathSelectionCard oder sowas denken. (Renaming geht in eclipse ganz einfach mit shift + alt + R wenn der cursor auf dem Klassennamen ist)